.github/actions/bump-version/package-lock.json

```diff
@@ -0,0 +1,3770 @@
+{
```

Comment:
A bummer, but if we can move all of this junk into its own public repo and then reuse the action across our repos that would be lovely.